src/main/cpp/t04_count.cpp

```diff
@@ -23,10 +23,21 @@
 
 #include "t04_count.h"
 #include <iostream>
+#include <string>
 
 
 using namespace std;
 
 int t04_count() {
+	string s;
+	int cnt = 1, i = 0;
+	getline(cin, s);
+	while ((int)(s.find(" ", i)) > -1) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Inefficient usage of string::find() in condition; string::starts_with() could be faster.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513059)